pkg/server/version.go

```diff
@@ -17,29 +17,30 @@ limitations under the License.
 package server
 
 import (
-	"fmt"
-
 	"golang.org/x/net/context"
 	"k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+
+	"github.com/kubernetes-incubator/cri-containerd/pkg/version"
 )
 
 const (
-	containerName        = "containerd"
+	// For now, containerd and runc are bundled with cri-containerd, cri-containerd
+	// version is more important to us.
+	// TODO(random-liu): Figure out how to package cri-containerd and containerd,
+	// and how to version it. We still prefer calling the container runtime "containerd",
+	// but we care both the cri-containerd version and containerd version.
+	containerName        = "cri-containerd"
```

Comment:
Maybe overload the containerName

if c.config.ContainerdConfig.RuntimeEngine == "" use "runc"

containerName = ... "cri-containerd; containerd; %s", runtimeengine

maybe 
containerName = ... "cri-containerd; containerd %v; %s %v", resp.Version, runtimeengine, runc.version

Reply:
We need a better way to fix this, only override the runtime name is not enough.